### PR TITLE
[Issue #1659] Write a script that copies csvs from s3 to the db + script to generate them

### LINKED ISSUE #1659: [Task]: Populate the Search API with seeded data in dev
### Summary

Populate API with factory seeded data (similar to how we can do it locally)

This work is blocking Usability Testing, and frontend connecting to dev and showing data. Staging will be done later

### Acceptance criteria

- API is seeded with mock data for dev

### PR DESCRIPTION
**Pull request by chouinar**
**tests / linting will fail until https://github.com/HHS/simpler-grants-gov/pull/1743 is merged**

## Summary
Fixes #1659

## Changes proposed
**this is temporary code and will be deleted after we run it - these are onetime scripts and are deliberately a bit hacky**

Adds two scripts:
* One that takes csvs extracts of the Oracle database tables and does transformations to make csvs we can import into our database
* One that takes the output of the first script and uploads them into our database

## Context for reviewers
As the transformation work is just starting, we want to unblock testing of the front-end UI by getting data into our dev environment. This approach is basically a hacky one-time shortcut to get data uploaded so that we have time to build the actual transformation process. Additionally, we using prod data so that it's realistic for testing purposes.

The unit test I setup for the load script just exists to verify the tools work if the input dataset is on disk (as I did locally) or via S3 (as it will actually be run).

## Additional information
Testing this requires downloading data from the real production database, which I think only I have access to at the moment. I did actually build this out / test it using that data locally. You can also use the test files I uploaded for the unit test to verify the behavior of the second part of the script.

To run it:
```
poetry run python tests/util/convert_oracle_csvs_to_postgres.py --directory <whatever directory your csvs are in>

poetry run flask task import-opportunity-csvs --input-folder <same directory as before>
```

Note this will not work at the moment due to needing a change from https://github.com/HHS/simpler-grants-gov/pull/1743

### CODE REVIEW

**Review by rylew1 — APPROVED** [review]

**Comment by rylew1** [comment]
Can you fix the lint error before merging please 🙏

**Review by rylew1 — DISMISSED** [review]
Just rubber stamping to keep things moving. Would it make sense to have a TODO comment and ticket ready here to clean some this stuff up later?

**Comment by chouinar** [comment]
@rylew1 

Lint error was due to needing the fix from the other PR, need another review as I upgraded a package version as I was seeing consistent failing due to an anchore scan for gunicorn: https://github.com/HHS/simpler-grants-gov/actions/runs/8743341824/job/23993763814

File: api/src/task/opportunities/import_opportunity_csvs.py
Comment (jamesbursa) [inline api/src/task/opportunities/import_opportunity_csvs.py:55]: Does this do anything? It's not being called, since it's missing `()`. Maybe DictReader gets the fieldnames without it.

Comment (chouinar) [reply api/src/task/opportunities/import_opportunity_csvs.py:55]: I think you're right. I had added it as I thought I remembered needing to read at least one line for dict reader to populate the field names, but removing it, it still works.

Looks like fieldnames literally does this in the property (although it looks like its done it for 16+ years, so no idea what I was remembering):
https://github.com/python/cpython/blob/main/Lib/csv.py#L160

**Review by coilysiren — APPROVED** [review]
looks like csv imports to me! 👍🏼

**Pull request merged**
